### DISCUSSION TITLE
Add Laadpaal

### DISCUSSION THREAD

**Discussion by robnieuw** [fsaris/home-assistant-zonneplan-one#93] [Ideas]
If you have a laadpaal from zonneplan you get extra items in the App. Like when you have loaded your car, loadHostory. Its olso posible to start loading where the price is low. Is it posible to add this to home Assistant app.
I love what it is now, but maybe you can make t better.
![IMG_3921](https://github.com/fsaris/home-assistant-zonneplan-one/assets/117859329/036e543b-1f31-44c0-b52c-f013aeae8fee)
![IMG_3920](https://github.com/fsaris/home-assistant-zonneplan-one/assets/117859329/66d57b43-9a29-49d1-b260-1af7907520bd)

**Comment by fsaris** [comment]
This is what you can setup yourself in HA. You could use the data to create a automation that starts loading when car is plugged and price is low (maybe even use the calendar features of HA).

The history (data) should already be visible in HA if you enabled the sensors (state, energy delivered session , next start, etc).

Scheduling a charge from HA in the Zonneplan app is out-of-scope IMO. 

Or you use the features from Zonneplan for this. Or your HA is in the lead and triggers a "start charge".
